requirements/image.txt

```diff
@@ -3,5 +3,5 @@
 
 scipy >1.0.0, <1.11.0
 torchvision >=0.8, <=0.15.2
-torch-fidelity <=0.3.0
+torch-fidelity @ git+https://github.com/toshas/torch-fidelity@master
```

Comment:
note that this req. will not be listed in package dependency if installed from PyPI

Reply:
Yes, but what do you have in mind, even prevent PR to be merged with this dependency?

Reply:
ref: https://github.com/toshas/torch-fidelity/issues/53
cc: @JustinGoheen